src/Components/ReconstructionCard.jsx

```diff
@@ -26,17 +26,24 @@ class ReconstructionCard extends React.Component {
         <div className='reconstruction-card-details'>
           <div className='card-details'>
             <div>
-              <h5 className='reconstruction-card-title'>
-                Project{this.props.reconstruction.name}
+              <h5
+                className={this.props.showCreator ? '' : 'heading-showCreator'}
```

Comment:
Name class appropriately.
```suggestion
                className={!this.props.showCreator ? 'heading-creator-hidden' : ''}
```